packages/web/app/src/components/layouts/organization.tsx

```diff
@@ -219,7 +220,16 @@ export function OrganizationLayout({
             <RateLimitWarn organization={currentOrganization} />
           </>
         ) : null}
-        <div className={className}>{children}</div>
+
+        {currentOrganization ? (
+          <div className={className}>{children}</div>
+        ) : (
+          <NotFoundContent
+            heading="Organization not found"
+            subheading="Use the empty dropdown in the header to select an organization to which you have access."
+            includeBackButton={false}
+          />
+        )}
```

Comment:
Does this change work for users with `"users"."is_admin"` being set to `true` (hive "admin"), being able to see organizations they are not a member of?